### PR TITLE
add export funcs for autoround and fix incorrect hype-parameters in example

### PR DESCRIPTION
**Pull request by WeiweiZhang1**
## Type of Change

feature
API not changed

### CODE REVIEW

**Review by xin3he — APPROVED** [review]
Please delete the commented code

File: neural_compressor/adaptor/torch_utils/autoround/autoround.py [hunk 1]
```diff
@@ -142,11 +142,13 @@ def quant_weight(weight, num_bits=4, group_size=-1, scheme="asym", v=0, min_scal
         weight = weight.reshape(-1, group_size)
         if isinstance(v, torch.Tensor):
             v = v.reshape(-1, group_size)
-
         weight, scale, zp = quant_weight_actor(
             weight, num_bits, scheme=scheme, v=v, min_scale=min_scale, max_scale=max_scale
         )
         weight = weight.reshape(orig_shape)
+        scale = scale.reshape(orig_shape[0], -1)  # TODO validating the feasibility on conv1d
```
Comment (wenhuach21) [inline neural_compressor/adaptor/torch_utils/autoround/autoround.py:149]: this reshape may have bugs for different layer type

File: neural_compressor/adaptor/torch_utils/autoround/autoround.py [hunk 2]
```diff
@@ -160,11 +162,49 @@ def quant_weight(weight, num_bits=4, group_size=-1, scheme="asym", v=0, min_scal
             weight_new, num_bits, scheme=scheme, v=v, min_scale=min_scale, max_scale=max_scale
         )
         weight_new = weight_new.reshape(orig_shape[0], -1)
-
+        scale = scale.reshape(orig_shape[0], -1)
```
Comment (wenhuach21) [inline neural_compressor/adaptor/torch_utils/autoround/autoround.py:165]: better add a function to do the reshape based on the layer type

File: neural_compressor/adaptor/torch_utils/autoround/model_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+        assert False, "Got an unknown version of torch: {}".format(e)
+    version = Version(torch_version)
+    return version
+
+
+PT_VERSION = get_torch_version().release
+
+
+class WeightOnlyLinear(torch.nn.Module):
+    def __init__(
```
Comment (wenhuach21) [inline neural_compressor/adaptor/torch_utils/autoround/model_wrapper.py:77]: @xin3he  Will you support transformer.conv1d later?

File: neural_compressor/adaptor/torch_utils/autoround/export.py [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            1: compression_dim: weight = 1, zeros = 0 and both are transposed.
+            2: zeros -= 1 before compression. Why we need it?
+            3: g_idx: use same number for one group instead of recording the channel order.
+            4. parameter name changed, such as 'packed_weight' -> 'qweight'.
+            5. zeros is always needed even for sym.
+    """
+    from .autoround import get_module, quant_weight_w_scale, set_module
+    from .model_wrapper import WeightOnlyLinear
+
+    compressed_model = copy.deepcopy(model)
```
Comment (wenhuach21) [inline neural_compressor/adaptor/torch_utils/autoround/export.py:65]: @xin3he  is there any way to avoid the deepcopy?

**Review by wenhuach21 — APPROVED** [review]

**Pull request merged**